### ISSUE TITLE
Stray file Makefile.am in rebar3 release

### ISSUE THREAD

**Issue by leoliu** [klacke/yaws#368]
I now use Yaws as a rebar3 dep and after building a release the `yaws-2.0.6` directory looks like this:

```
├── ebin
├── include
└── priv
```

all these subdirectories have a file `Makefile.am` which is unusual.

The issue can be reproduced with branch `master` or `rebar3-support`.

**Comment by vinoski** [comment]
Running `rebar3 release` doesn't work for me &mdash; I assume because of the lack of a `relx` section in `rebar.config` &mdash; but running `rebar3 compile` partly duplicates this problem, as after it finishes I see a couple `Makefile.am` files under the `_build` directory. I don't see any way to avoid having `rebar3` copy those files, but a post hook like the one below added to `rebar.config` can remove them:

    {post_hooks, [{"(linux|solaris|freebsd|darwin)", compile,
                   "find $REBAR_BUILD_DIR -name Makefile.am -exec rm -f {} \\;"}]}.

**Comment by leoliu** [comment]
I just noticed these files are in the repo i.e. version-controlled. So maybe ignore this issue for now. It's harmless anyway.

**Comment by vinoski** [comment]
Fair enough, but for now I've pushed a commit to run `find` over the build area after compilation as mentioned earlier in this issue.

**Comment by leoliu** [comment]
That could work. But...

is it safe to run it inside `_build`? It will make uncommitted/untracked changes to deps. Ideally it should only be run inside the `rel` directory or, even better, `yaws-VERSION` directory in the `rel` directory.

**Comment by vinoski** [comment]
I convinced myself a bit ago that the `find` addition wasn't really helpful, so I just pushed another change to remove it.
